rtengine/dcraw.cc

```diff
@@ -7113,8 +7113,8 @@ void CLASS apply_tiff()
     (unsigned)tiff_ifd[i].bps < 33 && (unsigned)tiff_ifd[i].samples < 13 && // RT
 	 ns && ((ns > os && (ties = 1)) ||
 		(ns == os && shot_select == ties++))) {
-      raw_width     = tiff_ifd[i].width;
-      raw_height    = tiff_ifd[i].height;
+      raw_width     = read_crop.complete ? read_crop.width : tiff_ifd[i].width;   // RT
+      raw_height    = read_crop.complete ? read_crop.height : tiff_ifd[i].height; // RT
```

Comment:
I think the raw dimensions are supposed to exclude the crop, which would make this unnecessary.